managed/ui/src/components/universes/UniverseForm/UniverseFormContainer.js

```diff
@@ -521,7 +526,8 @@ const universeForm = reduxForm({
   form: 'UniverseForm',
   validate,
   asyncValidate,
-  fields: formFieldNames
+  fields: formFieldNames,
+  asyncChangeFields: ['primary.universeName', 'async.universeName']
```

Comment:
Is this necessary?

Reply:
This is required to make async validate function call for specified fields